### DISCUSSION TITLE
Best way to restart Davinic after installing fuse

### DISCUSSION THREAD

**Discussion by rne1223** [nmbr73/Shadertoys#23] [Ideas]
This might not be a good idea, but I was thinking that maybe we can add a prompt to restart Davinci Resolve after installing the fuse. Something to ask the user "Would you like to restart Davinic Resolve now?" And the user has to select "yes" or "no".

The reason that I say that this might not be a good idea is because you might have to go OS level to kill and restart DR. Maybe there is a command that allows for this to happen. I will investigate.

**Comment by nmbr73** [comment]
Don't know much about Lua, but in the installer you see that there is some interface to the OS (I used `os.exit()` and afaik you can do e.g. an `os.execute()` which should open the door to pretty much everything). But retrieving the PID of the parent process (as Resolve initiates the Lua script), sending it sigterm or a kill and doing this in an OS independent way ... I have some doubts - but I'am curious what you find out.

**Comment by rne1223** [comment]
While reading the talk between Danelll and AndrewHazelden on the server I found this information: 
![image](https://user-images.githubusercontent.com/429191/109871293-ea336800-7c1f-11eb-9e3c-cc843a9eebf1.png)

I will investigate further once I get a chance.
